src/lib/create-model/validate.js

```diff
@@ -0,0 +1,35 @@
+const { get } = require('@0ti.me/tiny-pfp');
+const {
+  JSON_SELECTORS: { POOL, TABLE_NAME },
+} = require('../constants');
+
+module.exports = (context, config) => (model) => {
+  const pool = get(context, POOL, false);
+  const tableName = get(config, TABLE_NAME, false);
+
+  if (!pool) {
+    throw new Error(
+      `Could not find a pool on the context object with keys: "${Object.keys(
+        context,
+      )}"`,
+    );
+  }
+
+  if (!pool.query) {
+    throw new Error(`pool.query is missing ${Object.keys(pool)}`);
```

Comment:
Object.keys? Just the property names, yeah. It's good for introspection into what's going wrong. I assume if this ever happened it would be because something else got set to the `POOL` nested key.